src/otx/cli/manager/config_manager.py

```diff
@@ -43,6 +43,7 @@
     "ACTION_CLASSIFICATION": "Custom_Action_Classification_X3D",
     "ACTION_DETECTION": "Custom_Action_Detection_X3D_FAST_RCNN",
     "VISUAL_PROMPTING": "Visual_Prompting_SAM_ViT_B",
+    "ANOMALY": "otx_anomaly_padim",
```

Comment:
this is defined here but never used anywhere. Do you think if this is needed? Or the other model template ids updated?